metal/datasource_metal_device.go

```diff
@@ -93,11 +93,7 @@ func dataSourceMetalDevice() *schema.Resource {
 				Computed: true,
 			},
 			"storage": {
-				Type: schema.TypeString,
-				StateFunc: func(v interface{}) string {
-					s, _ := structure.NormalizeJsonString(v)
```

Comment:
We should make an issue to convert this from JSON to a structured value. I think we should make this match the CPR storage value submitted to the API, if possible.

Reply:
https://github.com/equinix/terraform-provider-metal/issues/115